build/package/.atlascli.goreleaser.yml

```diff
@@ -70,6 +70,15 @@ nfpms:
   formats: [deb,rpm]
   replacements:
     amd64: x86_64
+  contents:
+    - src: ./build/package/rpm/package_source
+      dst: /etc/atlascli/package_source
```

Comment:
instead of creating a file in a random folder could this reuse the config folder of the CLI?

Reply:
and if the config folder can't be used does uninstalling the CLI also removes this file?

Reply:
I can't use config folder because that is per user I need a global folder

Reply:
uninstalling also removes this file